docs/hello_world.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Hello, world!
+description: Your first steps with Prefect – learn how to create a basic flow and understand core concepts.
+icon: play
```

Comment:
these don't appear to be recently generated so either:
- we add a pre-commit hook to generate them and commit them
- or we gitignore them